amazons3.admin.inc

```diff
@@ -6,18 +6,38 @@
 function amazons3_admin() {
   $form = array();
 
+  $form['amazons3_use_environment_iam'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('I am using environment variables or IAM roles to manage my key/secret.'),
```

Comment:
I don't think other Drupal admin pages use "I am..." in the first person for text. How about `Use environment variables or IAM roles to manage credentials`?